a

kreszu opened the pull request:
b

KevinSmarts commented:
This is unlikely to be merged as kingofsweden is no longer involved in the
project.
The main repo is https://git.catrenelle.com/Kevin_Smarts/glife now.

Commit:
[changed] Added the possibilty for Sveta to take driving lesson if owning a fake passport

Commit:
[changed] Set name to the fake one instead of the real one in foto album for realism

Commit:
[fixed] Missing declaration of fake passport for driving license

Commit:
[changed] Add bodysuit to the status window in the toolbar

hornguy6 committed:
[added] converstion from booty calls into a more all purpose type of event

Commit:
[fixed] Correction of string on screen for transparent clothing photshoot

KevinSmarts committed:
Merge branch 'sources-changes' of StephanFuchs/glife into master

[fixed]Spelling Fix & Index Fix for $npc_notes

Commit:
[fixed] I don't knbow how to code.

Commit:
[fixed] Biodad is not mom. Also it don't make sense that the father of you baby doesn't like you prevent you from telling Katja that they are the father.

Commit:
Revert "[fixed] Missing declaration of fake passport for driving license"

This reverts commit 77df53a35e2baccbb695647c88534f51b94e3b63.

Commit:
Revert "[changed] Added the possibilty for Sveta to take driving lesson if owning a fake passport"

This reverts commit cc86534d872e5419fdb843c3465d98ba82f464f4.

Commit:
Merge branch 'master' of https://git.tfgames.site/Kevin_Smarts/glife

Commit:
[added] New attibutes for training "bodybuilder" strength and for a bigger butt. Changed the apperance calculation and body picture selection to take the new strength pluss varibale into account.

hornguy6 committed:
[fixed] original hookup setup situations now correctly send to the new files

hornguy6 committed:
[fixed] booty call date planning was using wrong variables

hornguy6 committed:
[added] various small things
[fixed] various small things

KevinSmarts committed:
Merge branch 'master' of jeep91/glife into master

[fixed] Wrong variables for foto events
[added] Display option for bodysuit in additional stat window

Commit:
[fixed] Cut and paste mistake made everybody much mor ugly,

MylesCroft committed:
The Kevin Will Hate Me update - Part I

Numerous QoL, code clean up and continued migration to new geoloc function in files.
There be dragons: Be wary, first draft of Natasha's quest overhaul with BBZ's new path. Bugs likely abound and it will break existing saves, initial draft is for the love path with corruption to follow.
Thousands separator added to numerous files - Thanks to Rachels for pointing out existing codebase to re-use.
Added lipbalm to handbag when you own a compact mirror.
Added minor dialogue scenes to Pav Pharmacy - Thanks to BBZ for writing the content.
Minor salon QoL bug fixes.
Sleep is set 100 on wake up.
Computer acts now set up so defaults to exit.
Simplified Niko fame calls as per BGK advice.

Commit:
[changed] some sonia stuff by nutluck.

KevinSmarts committed:
[Test]

KevinSmarts committed:
Merge remote-tracking branch 'Hornguy6/master'

Commit:
[added] Some music store stuff by nutluck.

hornguy6 committed:
[fixed] forgot to remove duplicates - the old booty_call folder is still in there as a backup, currently holding off deleting it

Commit:
[added] Making sure you don't have honework when school is over.

Commit:
[fiexd] More safe guards to the setting of grades in the intro.

KevinSmarts committed:
Merge remote-tracking branch 'Hornguy6/master'

KevinSmarts committed:
Merge remote-tracking branch 'origin/master'

Commit:
[fixed] Temporary variable for calculating groming penelties to appearence not reset properly.

Commit:
[fixed] Not updated to correct rage of pcs_skin

Commit:
[changed] Small update on the City Nightclub. A full bar with menu, nomore prostitution but you can be picked up by a couple on the dance floor.

JDGBOLT committed:
Fix variable timeTresh using a non-existent variable hypnoType vs the intended hypnoTime

timeTresh which is used for the missCum comparisons was using a variable
that didn't exist, hypnoType, whereas hypnoTime seems to be the intended
comparison as it is capped in the line before, as well as it's name is
timeT(h)resh, and a max value of 15 would make the minimum value for
timeTresh 16 rather than the max 76.

KevinSmarts committed:
Merge branch 'hypno-threshold-fix' of Unstable.Egghead/glife into master

[Fixed] non-existent variable hypnoType vs the intended hypnoTime

KevinSmarts committed:
[Fixed] This variable should never have been, thanks Unstable Egghead for pointing out its loney and pointless existance

KevinSmarts committed:
Merge remote-tracking branch 'origin/master'

Commit:
[fixed] Apprently Feofan loves that you flash other people that don't care. Also change it to the proper gs.

Commit:
[changed] Chnge some instrmusic_lvl checks to pcs_instrmusic, since that is skill that ells how well you play, and added exp gain to playing with nush.

Commit:
Fixed missing image which is already present in media download

Commit:
Fix various typos

Commit:
Fix more typos in Anna story arc

Commit:
Fix more typos in Anna's story arc

Commit:
Fix some more typos in Anna's arc

Commit:
Fix typos in Leonid arc

Commit:
One more typo

hornguy6 committed:
[changed] many things about the sex_ev files
[fixed] booty_call texts firing?

Commit:
Some small grammar edits

Commit:
Fix Grammer in random encounter.

Commit:
[Text Edit] Rolan's events Capitals, text from variable names.

hornguy6 committed:
[fixed] missing variable setting might have caused a problem

KevinSmarts committed:
[Added] Filters for bras, finally

KevinSmarts committed:
[fixed] Wrong number in ARGS leadin to no card charge when buying guitar

KevinSmarts committed:
[fixed] No cum was being added from these sex events

KevinSmarts committed:
[fixed] Condoms in cheat menu to show all types and should fix problem with count being wrong

KevinSmarts committed:
[changed] experience gains from chess increased depending on level of opponant and added fame on same scale

KevinSmarts committed:
Merge remote-tracking branch 'Hornguy6/master'

KevinSmarts committed:
[fixed] Bras were not being sorted by quality in error

KevinSmarts committed:
Merge remote-tracking branch 'Hooded_Silence/master'

KevinSmarts committed:
[revert] Changes to Nat from HS reversed by request as incomplete

KevinSmarts committed:
[added] filters for panties and bodysuits, with cleanup for bras

Commit:
[fixed] Fixing nutlucks nutlucking

KevinSmarts committed:
Merge branch 'master' of hotmike/glife into master

[fixed] missing image in day time event in park. Image is present in media download.
[fixed] various typos/grammar updates in Anna's and Leonid's story arc

KevinSmarts committed:
[fixed] Names for 2 stores showing on quality sort in error

KevinSmarts committed:
[fixed] Sleep was been held in suspension due to wrong sign and added daystart check to fix early next morning sleep deg

Commit:
Rolan Grammar

Commit:
Merge branch 'hotmike_1'

Commit:
Merge remote-tracking branch 'upstream/master'

KevinSmarts committed:
[fixed] Scandalicious everywhere!!

Commit:
Rewrite the chance code for Rolan. Now the chance should be 1/3 for a panty steal to start.

Update text again. Should all be legible English now.

KevinSmarts committed:
[fixed] Typos found by analyser

KevinSmarts committed:
[fixed] Correction for living at home check in your bedroom at parent's house

Commit:
Merge remote-tracking branch 'upstream/master'

Commit:
[Fix] missing space, reported by yanzaqw1

Commit:
[changed/fixed] Fixed the date of the last day of school.

KevinSmarts committed:
Merge remote-tracking branch 'HotMike/master'

Commit:
[fixed] Changed the SchoolAtestat = 0 to $home['current'] = 'parents_home' so you can actaully eat at home whne still living there.

Commit:
[fixed] Mother should also use the correct day for graduation now.

hornguy6 committed:
[fixed] using shoe variables for coats instead of coat variables for coats

Commit:
Text edit. Fixed my own typo

Commit:
Add some explanation, fix typos

Commit:
Add some comments on variable usage.

Commit:
Added bare files from nutluck

Commit:
Added file from nutluck. Already changed some of code and did some text edits

Commit:
Text edits mainly

Commit:
Added a location for anushapt_city

Add the vars that are set at entry of the apartment. But will need a dynamic calculation for those. Notetoself: find out where that is set.

Commit:
[added] - new 2023 splashes

KevinSmarts committed:
Merge remote-tracking branch 'HotMike/master'

KevinSmarts committed:
Merge remote-tracking branch 'Anya/master'

KevinSmarts committed:
[fixed] Eye colour couldn't decide if its gray or grey
[changed] can see eye colour instead of face image in intro customisation for eyes section, if Anya will add the images in next pack (which she might not, we are negotiating)

KevinSmarts committed:
[fixed] "text edits and line breaks added. The main change is changing the person that stops you in the hall from Sonia to a teacher, since Sonia no longer makes sense with the changes to how we make her fall now." - nutluck

Commit:
Add a return location so we can use this chat from multiple locations.

Commit:
Add return location

Commit:
format code

Commit:
Add return location

Commit:
Add the current state. Adomir's room is still broken. Maybe someone can see what goes wrong where?

KevinSmarts committed:
[added] New butt exercises, images in next image pack

KevinSmarts committed:
[changed] Butt training gains merged into exercise gosub to make it cleaner, thanks bg!

KevinSmarts committed:
[added] updater to hopefully fix fuckbuddy bug in old saves

Commit:
Proofreading. Nuff said.

KevinSmarts committed:
[changed] Moved gym exercise images to more logical location under instruction from Anya

KevinSmarts committed:
Merge remote-tracking branch 'Vengeance_11/master'

Commit:
Fix missing quotes, found by Kevin Smarts

Commit:
Fix the unknown action - Thanks to hornguy6

Commit:
[fixed] image address to match existing folder

Commit:
Text edit: typos and grammar in Katja.

KevinSmarts committed:
Merge remote-tracking branch 'HotMike/master'

KevinSmarts committed:
[fixed] Attributes for school skirts and some of the messages for them were wrong and needed re-evaluation

Commit:
Use the locat (location) array and stories NPC locations in there.

Commit:
Add quest log

Edit text Fix headshot IDs
Add more location IDs

KevinSmarts committed:
[fixed] group bonus from "rebel" uniform to Gopnik added to stairwell chat and fixed inhib check in wardrobe

KevinSmarts committed:
[fixed] 3 amigo beer time makes the boys like you now

KevinSmarts committed:
[fixed] Screen not being cleared causing actions to remain

KevinSmarts committed:
[fixed] added thje word squat to the 5th column

KevinSmarts committed:
[fixed] typo

KevinSmarts committed:
[fixed] correction for heavy weight use

Commit:
You will know about Nus' apartment when you first meet her at the coffee Hole. Additional: she needs to like you.

Commit:
Add TODOs

hornguy6 committed:
[fixed] consistency and dead ends
[changed] blocked some WIP stuff
[added] more dialogue options

KevinSmarts committed:
[fixed] some old style cum calls removed and tidied things up so that it is a bit easier to follow

KevinSmarts committed:
[fixed] Very old style cum calls updated and so gosubs I made into gotos in error in last commit

Commit:
Looking for the time error. When days +1, there is an no [end] found error occuring

Add todos

KevinSmarts committed:
[text edit] missing word

KevinSmarts committed:
Merge remote-tracking branch 'Hornguy6/master'

KevinSmarts committed:
[fixed] timings for Squats in exercise sets all set to show first set

KevinSmarts committed:
[added] Option to change into and out of sports clothing directly in places where you can exercise and access changing room

KevinSmarts committed:
[fixed] Returning to wrong place when using default sports shortcut and wrong variable in array breaking the Too Risque only button

hornguy6 committed:
[fixed] missing word

KevinSmarts committed:
Merge remote-tracking branch 'Hornguy6/master'

KevinSmarts committed:
[fixed] number separaters and nipple/lip confusion resolved from Hooded Silence

Commit:
First commit of a separated schedule. Mainly showcasing.

Commit:
Need to use '' even in comments. I adore QSP

Commit:
One needs to use '' even in comments

Commit:
Grammer in comments

Commit:
Include Valentin in the schedule

Commit:
[changed] The redoing of stamina to actaully mean something. I'm waithing for the tears of the players.

Commit:
[changed/added] Clean up os stat means this have to be pushed now. Includes a lot of changes and addition for the upcomming Katja nigthclub content.

Commit:
Add standard Radomir schedule. Should be adjusted probably, need input of writer.

KevinSmarts committed:
[changed] updated/added checks to prevent exercise based on changes to stamina

Commit:
Current status. Still need to finetune the locations.

KevinSmarts committed:
[changed] Settings for new official dev release with version history

KevinSmarts committed:
[fixed] Missing end and got the exercise check for manual selection the wrong way around

MylesCroft committed:
QoL update - non-combat spells should now display immediately, maybe - credit BGK

KevinSmarts committed:
[fixed] Could do manual exercises if stamina or energy was high enough, it should be both

netuttki committed:
Merge remote-tracking branch 'upstream/master'

KevinSmarts committed:
Merge remote-tracking branch 'Hooded_Silence/master'

hornguy6 committed:
[fixed] calling wrong file name

Commit:
[fixed] correction of a correction

hornguy6 committed:
[added] can now pillow talk while taking a break in the middle of sex

hornguy6 committed:
[changed] text edits, bug fixes, and general reorganisation

Commit:
Merge remote-tracking branch 'upstream/master'

Sepishow committed:
[fixed] change to sport clothes and back function

Also added check for default sports clothing set being deleted.

Commit:
Add a kiss counter for Anushka to make the sex progress more logical.

Sepishow committed:
[added] new basic event to tell mother that you were raped

Purpose: to not make mother angry for loosing your virginity to a raper.

KevinSmarts committed:
Merge remote-tracking branch 'Hornguy6/master'

Commit:
Some comments while browsing files.

Commit:
[fixed] two events had the wrong image code #blamenutluck

KevinSmarts committed:
[fixed] pain should be in the head as hooded Silence pointed out

Commit:
[changed] Taking a nap now counts as sleeping for resource use/recovery.

KevinSmarts committed:
Merge remote-tracking branch 'Anya/master'

KevinSmarts committed:
Merge remote-tracking branch 'origin/master'

Commit:
Improve schedules. Add starter of journal function

Commit:
Merge branch 'nutluck_230508_city_schedule' into nutluck_230508

Commit:
Add basics for journal. Now the journal file will make a call for a function that generates the text. Only the quest files need to be updated then.

Commit:
Comment updates

Commit:
Merge branch 'nutluck_230508'

# Conflicts:
#	locations/stat.qsrc
Solved

Commit:
Add a short sentence when you talk to a teacher at lunch in school. Gives some feedback to the player.

KevinSmarts committed:
Merge remote-tracking branch 'sepis/master'

KevinSmarts committed:
Merge remote-tracking branch 'HotMike/master'

KevinSmarts committed:
[fixed] Return to platform and not train hall after chat with Bella on the platform now

KevinSmarts committed:
[fixed] merged the two same pieces for school uniform text in clothing files into one and made the line that was not 100% clear less ambiguous

KevinSmarts committed:
[fixed] Calculations for exercise cost when viewing routine updated

Commit:
[fixed] Grades should alos be upgrade in the exam week before you check them. Spotted by Genjuroooooo

hornguy6 committed:
[fixed] dead end

hornguy6 committed:
[fixed] missing $s

hornguy6 committed:
[fixed] calling wrong file

hornguy6 committed:
[added] failsafe option with bug report message

hornguy6 committed:
[fixed] added missing flags

MylesCroft committed:
QoL: chats with your brother will now add to relationship points - once per day.

KevinSmarts committed:
Merge remote-tracking branch 'Hornguy6/master'

KevinSmarts committed:
Merge remote-tracking branch 'Hooded_Silence/master'

Commit:
[fixed] Some cut and paste mistakes. Spotted by Steuchs

KevinSmarts committed:
[fixed] no action for this scene if you have to shave

MylesCroft committed:
Proofing: Fixed type - spotted by Astrial.

KevinSmarts committed:
[fixed] Stamina cost calculation was bad, bg pointed out my error

KevinSmarts committed:
Merge remote-tracking branch 'Hooded_Silence/master'

KevinSmarts committed:
[fixed] Sleep variables set in wrong place to prevent you starting day with 96 sleep if you set alarm for quarter to the hour or less

MylesCroft committed:
The great Liam name change to Pushkin. File name change to make the location easier to find under Pushkin which is how it's referred in-game, all image names remain the same as do their paths. Liamel Café remains the same in display name, but filename has changed. Location transition testing completed, but quests require further testing to ensure no bugs.

KevinSmarts committed:
[fixed] Dancer was too inhibited and is not most uninhibited start. Should be able to wear her leotard now too

netuttki committed:
Merge remote-tracking branch 'upstream/master'

netuttki committed:
Changed the volleyball and Starlets pregnancy check

Instead of checking `pregtime >= 90` the reaction will depend on the result of `pregnancyVisibility` same way it is handled with family members.

KevinSmarts committed:
[fixed] another variable that needed adding for squats to maybe show the stamina use now, who knows?

KevinSmarts committed:
Merge remote-tracking branch 'Hooded_Silence/master'

KevinSmarts committed:
Merge remote-tracking branch 'Netuttki/master'

KevinSmarts committed:
[removed] A few pube options were to similar and the images were not that great so Anya condenced them

Commit:
[changed] Christina shower humiliation is now linear until her house unlocks, then resumes random

KevinSmarts committed:
[fixed] The Niko revenge did not fire if you asked for her help. now it will and invite to Anushka's apartment while meeting her at the coffee hole so it sounds a bit more like her both from nutluck

KevinSmarts committed:
[fixed] allow saving of 'nude' outfits

KevinSmarts committed:
[text edit] Making the rebel flagged uiniform clearer in its appeal

KevinSmarts committed:
[changed] Attribute for this uniform was wrong

KevinSmarts committed:
Merge remote-tracking branch 'Anya/master'

KevinSmarts committed:
[added] just a clarification for clothing description

hornguy6 committed:
[fixed] a lot of incorrect variable names

hornguy6 committed:
[changed] blocks breakfast if you say you're late for school

KevinSmarts committed:
[chaged] Just clarifying the reason for some uniforms failing to meet school uniform regs

KevinSmarts committed:
[changed] Price for Lusso bodysuits was a bit low

KevinSmarts committed:
[changed] Balanced the price for these bodysuits

KevinSmarts committed:
[fixed] some starting clothing for school starts was too risque and uniforms were not all suitable. Changed the confirm action there to make it much clearer too

KevinSmarts committed:
[fixed] exercise for swimming in lakes not set in Pav and set wrong in city

KevinSmarts committed:
[changed] Move the latching in parent's bathroom to links as the actions were a bit overwhelming

KevinSmarts committed:
[changed] Arty Goth wasn't arty enough and her poetry is getting better

Commit:
[chenged] Some adjustment about how long it takes for you to walk home with Katja and some new respondse if you talk about mausterbation or lesbian sex and you have not tried while she has.

Commit:
Niko files given a much needed clean up

Commit:
Text edits

KevinSmarts committed:
[added] Makup tattoo can be removed in cheat menu now

KevinSmarts committed:
Merge remote-tracking branch 'Vengeance_11/master'

KevinSmarts committed:
[changed] Old dates completely redone and replaced by BBZ01 with code from me

Commit:
Porn studio tweaks

Commit:
[fixed] Unclosed string.

KevinSmarts committed:
[fixed] Prevent desire to go to therapist if stuck in sauna paying off debt

KevinSmarts committed:
[changed] Moved exit option in parent's bathroom to top of the action list. All must comply

KevinSmarts committed:
[fixed] Variable missing its postfix so it was using from another source in same file and converted to elseif structure

KevinSmarts committed:
[fixed] Varible update from nutluck

Commit:
Old 'cls' left over

KevinSmarts committed:
[fixed] Hungry, hungry ATM fixed by HS

Commit:
[fixed] Missing *cl & cla. Also place holder names for the cats. For that I blame the nutty one.

KevinSmarts committed:
Merge remote-tracking branch 'Vengeance_11/master'

KevinSmarts committed:
Merge remote-tracking branch 'origin/master'

KevinSmarts committed:
[fixes] Errors spotted by analyser

Commit:
Variable into array

KevinSmarts committed:
[fixed] Sveta's model name should be used here

KevinSmarts committed:
Merge remote-tracking branch 'origin/master'

Commit:
[fixed] Use proper variable insread of realtion higher thatn 0 to check that the player have met Kat, and some formating.

KevinSmarts committed:
[fixed] Code chunk reinstated to fix quest, thanks Dakkhon for posting about it

Commit:
[Added] Big Katja nightclub content. Plus system for Katja doing things with out the player being there (sex/drugs), and chnges to allow start of Katja''s quest in diffirent states of her sexual progression.

Commit:
Merge branch 'master' of https://git.tfgames.site/Kevin_Smarts/glife

MylesCroft committed:
Life flashes before your eyes edition:

Photography shop overhaul redux: Refactoring and re-organising content.
Events are now gs calls on demand rather than dynamic functions - means manually increasing the loop counter when adding jobs but reduces memory bloat.
Events should now fire properly and you actually get paid! All events go through a 'stage' initiator reducing complexity and have only one point of failure for bug testing.
All photography events are now in photography_work splitting the content away from shop_photography, this is to allow future photography content from other locations to go through the same file.
Fixed the variable conflict from stat_display and event display functions.
Replaced the photography experience counter with an array to count completed jobs - probably not needed due to the overhaul breaking everything!
Reduced the working hours to something a bit more reasonable 2-4, but initial wages are still low to allow for skill scaling and other future content.
Bug hunt: fixed the fixes that snitched.

Please report any issues to Discord for my attention.

KevinSmarts committed:
[added] Saveupdater file for Hooded

KevinSmarts committed:
Merge remote-tracking branch 'Hooded_Silence/master'

# Conflicts:
#	locations/saveupdater.qsrc

Commit:
Edits

KevinSmarts committed:
[fixed] Grammer for when cum spatters, doesn't affect when it covers or coats

hornguy6 committed:
[fixed] updated necessary conditions for failsafe message

Commit:
Merge remote-tracking branch 'upstream/master'

Commit:
Rewording Anushka journal. @Smh @Lordlrontoe

KevinSmarts committed:
Merge remote-tracking branch 'Hornguy6/master'

KevinSmarts committed:
Merge remote-tracking branch 'HotMike/master'

KevinSmarts committed:
[fixed] A couple of hypno variables not cleared in reset.

Commit:
[dixed] Missing text where Katja explains that she don't want a thresome if she is not horny.

KevinSmarts committed:
Merge remote-tracking branch 'Vengeance_11/master'

KevinSmarts committed:
Merge remote-tracking branch 'origin/master'

KevinSmarts committed:
[fixed] Typo spoted by R.

KevinSmarts committed:
[added] Comments to the school chats for the new sub catergories of uniform from BBZ01
